setup.py

```diff
@@ -92,6 +92,7 @@ def write_git_revision(package_name):
           "codepy>=2017.1",
           "colorama",
           "Mako",
+          "treelib",
```

Comment:
Instead of adding a dep. I should probably add a minimal tree implementation in pytools itself.

Reply:
(That, or just add it here... should be like 10 lines?) The license is also slightly mismatched, given that it's Apache.

Reply:
We have an in-house data structure now. It's immutable and has a better interface than the one in treelib. Resolving this.